gto-support-scarlet/src/main/java/com/tinder/scarlet/internal/utils/RuntimePlatformInternals.java

```diff
@@ -0,0 +1,12 @@
+package com.tinder.scarlet.internal.utils;
+
+import androidx.annotation.NonNull;
+
+class RuntimePlatformInternals {
+    /** @noinspection KotlinInternalInJava*/
+    @NonNull
+    @SuppressWarnings("MethodName")
+    static RuntimePlatform Default() {
+        return new RuntimePlatform.Default();
```

Comment:
`RuntimePlatform` is a Kotlin internal class in another library, we can still access it from Java to create the instance we need.